### PR TITLE
Amls 5125 - Conditional reveals not stacked correctly in Pre-app variation and amendment

### PR DESCRIPTION
**Pull request by Dandyboyo2**
Conditional reveals not stacked correctly in Pre-app variation and amendment

## How Has This Been Tested?
Manual Testing
Unit Tests passing

## Checklist
- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [?] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/businessmatching/updateservice/add/fit_and_proper.scala.html [hunk 1]
```diff
@@ -56,8 +56,6 @@
             )
         }
 
-        <p></p>
```
Comment (Dandyboyo2) [inline app/views/businessmatching/updateservice/add/fit_and_proper.scala.html]: Unnecessary?

File: app/views/businessmatching/updateservice/add/fit_and_proper.scala.html [hunk 2]
```diff
@@ -41,8 +41,8 @@
             heading = "businessmatching.updateservice.fitandproper.heading",
             section = "summary.updateservice",
             panel = false,
-            supportingContent = supportingContent
-
+            supportingContent = supportingContent,
+            classes = Seq("inline")
```
Comment (Dandyboyo2) [inline app/views/businessmatching/updateservice/add/fit_and_proper.scala.html]: Additional ratio button (outside of ticket) where the buttons were stacked vertically despite no conditional reveal, this is now corrected.

File: app/views/businessmatching/updateservice/add/trading_premises.scala.html [hunk 3]
```diff
@@ -37,7 +37,9 @@
             heading = Messages("businessmatching.updateservice.tradingpremises.heading", Messages(s"businessmatching.registerservices.servicename.lbl.$activity.phrased")),
             section = "summary.updateservice",
             panel = false,
-            errorGroupPlaceholder = Messages(s"businessmatching.registerservices.servicename.lbl.$activity.phrased")
+            errorGroupPlaceholder = Messages(s"businessmatching.registerservices.servicename.lbl.$activity.phrased"),
+            classes = Seq("inline")
+
```
Comment (matthewharris1) [inline app/views/businessmatching/updateservice/add/trading_premises.scala.html]: Remove whitespace?

Comment (Dandyboyo2) [reply app/views/businessmatching/updateservice/add/trading_premises.scala.html]: Cheers, made that update now.

File: app/views/businessmatching/updateservice/add/fit_and_proper.scala.html (hunk 1 above)
Comment (slavvco) [reply app/views/businessmatching/updateservice/add/fit_and_proper.scala.html]: if I remember well it's needed here

**Pull request merged**